catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -472,6 +474,20 @@ public Object visit(PropertyIsGreaterThanOrEqualTo filter, Object delegate) {
             return lessThanOrEqual.accept(this, delegate);
         }
 
+        //Special case for handling temporal queries
+        if (literal instanceof String && ((String) literal).matches(
+                "RELATIVE\\(P(?!$)(\\d+Y)?(\\d+M)?(\\d+W)?(\\d+D)?(T(?=\\d)(\\d+H)?(\\d+M)?(\\d+S)?)?\\)")) {
```

Comment:
When I was trying before, using that method didn't work for me. I'll give it another shot though